README.md

```diff
@@ -3,6 +3,8 @@
 This repository contains the current, and historical, JSON Schema
 specifications.
 
+It should be noted that the master branch of this repo does not constitute a currently released version.
+For draft 3 or 4, please see <https://tools.ietf.org/html/draft-zyp-json-schema-03> and <https://tools.ietf.org/html/draft-zyp-json-schema-04> respectivly
```

Comment:
Should we instead refer folks to this repository's tags to find released versions in the change history, and the web site to find links to the current drafts?  This PR is already out of date and it would be nice to only need to update the web site and not this README as well when a new draft is released.